pkg/controller/acmeorders/util_test.go

```diff
@@ -124,7 +124,7 @@ func TestChallengeSpecForAuthorization(t *testing.T) {
 				Challenges: []cmacme.ACMEChallenge{*acmeChallengeHTTP01},
 			},
 			expectedChallengeSpec: &cmacme.ChallengeSpec{
-				Type:    "http-01",
+				Type:    cmacme.ACMEChallengeTypeHTTP01,
```

Comment:
In v1 these types are all upper case strings. I decided to use the constant too, to avoid this problem if the type strings ever change.